view/frontend/templates/beforetopsearch.phtml

```diff
@@ -1,14 +1,16 @@
 <?php
```

Comment:
I think it would be great to rename the file from `beforetopsearch.phtml` to something more descriptive, like `commonjs.phtml`, wouldn't it?